packages/multi-select-combo-box/src/vaadin-multi-select-combo-box-internal-mixin.js

```diff
@@ -435,4 +441,13 @@ export const MultiSelectComboBoxInternalMixin = (superClass) =>
 
       super.clearCache();
     }
+
+    /** @private */
+    __onCustomValueSet(event) {
+      // Prevent setting custom value on input blur or outside click,
+      // so it can be only committed explicitly by pressing Enter.
+      if (this._ignoreCommitValue) {
+        event.preventDefault();
```

Comment:
Could it stop propagation already here? This might let us get rid of `if (event.defaultPrevented)` in the next file.

Reply:
Using `stopImmediatePropagation()` does help. Updated the PR.